components/Page_Layout/FutureEventCard.js

```diff
@@ -0,0 +1,22 @@
+import '../../css/FutureEventCardStyle.css';
```

Comment:
Download the extension Prettier, it’ll auto format the code for you whenever you save your file. Use prettier to all of the files in the PR. This ensures that the formatting of our code is consistent